semantics/executable-spec/src/Control/State/Transition/Extended.hs

```diff
@@ -170,6 +173,11 @@ class
 
   type BaseM a = Identity
 
+  type Event a :: Type
+  type Event a = TypeError ('Text "missing associated type for '" ':<>: 'ShowType (Event a) ':<>: 'Text "'")
```

Comment:
these default implementations are here to make everything compile without changing much for now